socialhome/streams/views.py

```diff
@@ -56,17 +56,18 @@ def dump_content(content):
             }
 
         return {
-            "contentList": [dump_content(content) for content in self.queryset[:self.paginate_by]],
+            "contentList": [dump_content(content) for content in self.get_queryset()[:self.paginate_by]],
             "currentBrowsingProfileId": getattr(request_user_profile, "id", ""),
             "streamName": stream_name,
             "isUserAuthentificated": bool(self.request.user.is_authenticated),
             "showAuthorBar": not stream_name.startswith("profile_"),
-            "translations": {
-                "stampedContent": {
-                    "h2": gettext("Public"),
-                    "p": gettext("Contains public content from around the network.")
-                }
-            }
+            "translations": {"stampedContent": self._get_stamped_content_translations()}
+        }
+
+    def _get_stamped_content_translations(self):
+        return {
+            "h2": gettext("Public"),
+            "p": gettext("Contains public content from around the network.")
```

Comment:
This way is ok for this PR but don't expand too much on it. We have to provide this content as a Vue component. The profile streams are way too complex to be done like this, they have user actions and stuff.

I think (after this PR) the stream stamped area should be a Vue component, one each for each stream type, and then we just decide which one to render depending on the stream name.